### DISCUSSION TITLE
MyAsus stopped working for driver updates Asus tuf f15

### DISCUSSION THREAD

**Discussion by RahulVivekNair** [seerge/g-helper#470] [Q&A] [answered]
I followed the instructions in the readme and installed g-helper and it works perfectly. The only issue I have is that MyAsus now shows "Please restart the system to ensure the functions work properly" and the bios and driver update tab is missing. This is the only thing i'm missing. Any service I should enable again just for the driver updates?

**Comment by seerge** [comment]
@RahulVivekNair you need all asus services running for that. So run [bloat.bat](https://raw.githubusercontent.com/seerge/g-helper/main/bloat.bat)

**Reply by RahulVivekNair** [reply]
Thanks for the reply,are all the services required or just specific ones?

**Comment by seerge** [comment, marked as answer]
@RahulVivekNair they are required for different sections of myasus :)  for updates you need "software manager" i think.
